### ISSUE TITLE
Missing i18n key

### ISSUE THREAD

**Issue by larsauswsw** [corona-warn-app/cwa-app-ios#5024]
![image](https://user-images.githubusercontent.com/2742747/214958001-a4476458-fbf9-4bc3-b53c-1a3cf1710037.png)

## Technical details

- Device name: iPhone 12
- iOS version: 16.3
- App version: 3.0.0 (18)

## Describe the bug
Incomplete localization.

## Steps to reproduce the issue
Install CWA. Try to add a Selbsttest.

## Expected behaviour
Missin i18n key MIN_TIME_SINCE_ONBOARDING

## Possible Fix
Add i18n key

**Comment by Ein-Tim** [comment]
`MIN_TIME_SINCE_ONBOARDING` is an error code, it is not intended to be localized.

**Comment by larsauswsw** [comment]
Why should I See an error code as customer?

**Comment by Ein-Tim** [comment]
@larsauswsw 

You can call the technical hotline and give them the error code and they will know exactly what the problem is.

- FYI: @flxschmidt removed the prefix "Fehlercode:" before the error code in https://github.com/corona-warn-app/cwa-app-ios/pull/4979.
Maybe it makes sense to add it back @dsarkar & @thomasaugsten?

**Comment by dsarkar** [comment]
@Ein-Tim I understand that it is intended that the word "Fehlercode" was removed. This is consistent across the app and the same for Android and iOS.

Here another example:

![image](https://user-images.githubusercontent.com/1409741/215046943-29062533-72c8-4ad9-984b-e45d1819dde6.png)

**Comment by dsarkar** [comment]
@larsauswsw Thanks for reporting. If your "Selbsttest" was positiv and you installed the Corona-Warn-App just now you can use the warning feature only after 48 h (current configuration). However, since you haven't had installed the app on your phone before your test, non of your contacts you have had in the days before your positive test will be warned.


Best wishes, DS

---
Corona-Warn-App Open Source Team

**Comment by Ein-Tim** [comment]
@dsarkar I understand that it was the intention to remove the word "Fehlercode", however, I guess @larsauswsw isn't the only one who could be confused by this. Maybe you can relay it back to the team so that they take it into re-consideration?

**Comment by larsauswsw** [comment]
Thank you for the explanation. But as you already mentioned I was a little bit confused and would better understand it with the prefix "Fehlercode".

**Comment by dsarkar** [comment]
@Ein-Tim For that, we would need to create a story/wishlist item. If you would like to create this I will mirror it together with the input form this issue here, however, I see little chance that it will be implemented since only recently this was discussed and decided that the word "Fehlercode" was not necessary in both iOS and Android apps.

**Comment by Ein-Tim** [comment]
@dsarkar 

I would leave it to @larsauswsw to [create a new issue in the cwa-wishlist](https://github.com/corona-warn-app/cwa-wishlist/issues/new/choose) repository.

**Comment by larswmh** [comment]
Thanks for your report @larsauswsw. As explained in the comments above by @dsarkar & @Ein-Tim, this is an error code that is not intended to be localized. Feel free to open an enhancement request in the cwa-wishlist repository, but keep in mind that this change was recently discussed. I will now close this bug issue.

---
Corona-Warn-App Open Source Team
